docs/api/events.md

```diff
@@ -0,0 +1,520 @@
+# iCalendar
+All types are iCalendar([RFC5545](https://tools.ietf.org/html/rfc5545) and [RFC7986](https://tools.ietf.org/html/rfc7986)) types: `Event`; `Journal`; `Todo`.
+iCalendar and [jCal](https://tools.ietf.org/html/rfc7265) formats are supported and can be obtained by using `text/calendar` or `application/calendar+json`, respectively, in the `Accept` header of requests. If the request made has `application/vdn.siren+json` on the `Accept` header then a in-house representation will be used. This representation still holds the same information as jCal, it simply is displayed in what the team considers a more convient format.
+
+iCalendar components support sub-components, however, the components being used(`Event`, `Journal` and `Todo`) cannot be nested in other components so there would never exist any subcomponents. As a consequence, the `subComponents` property of the objects has been ommitted.
+
+# Date Format
+Date formatting is done according to [RFC3339](https://tools.ietf.org/html/rfc3339) which is an Internet profile of the [ISO8601](https://www.iso.org/iso-8601-date-and-time-format.html).
+
+# `Event`
+An `Event`, as described [here](https://tools.ietf.org/html/rfc5545#section-3.6.1) describes an occurrence within a given time frame, such as an exam, a meeting or an appointment. It can also represent a repeating occurrence if the [`rrule`](https://tools.ietf.org/html/rfc5545#section-3.8.5.3) property is specified. __Not all properties mentioned in the RFC are supported by this API, only the ones mentioned below. All properties parameters are supported.__
```

Comment:
> as described here describes

I suggest changing to something along the lines of: "An Event represents/is(<- link here) an occ...

Reply:
I have "an Event, described here, represents [...]" in my local branch. Is that ok?